src/cdk-experimental/testing/component-harness.ts

```diff
@@ -267,7 +271,14 @@ export class HarnessPredicate<T extends ComponentHarness> {
   private _predicates: AsyncPredicate<T>[] = [];
   private _descriptions: string[] = [];
 
-  constructor(public harnessType: ComponentHarnessConstructor<T>) {}
+  constructor(public harnessType: ComponentHarnessConstructor<T>, options: BaseHarnessFilters) {
```

Comment:
Any reason why we make the options required here? I could imagine having `{}` as the default.

Reply:
Fair enough 😄